api/src/pcapi/scripts/algolia_indexing/indexing.py

```diff
@@ -25,6 +26,40 @@ def batch_indexing_offers_in_algolia_from_database(
         page += 1
 
 
+def batch_indexing_collective_offers_in_algolia_from_database(
+    ending_page: int = None, limit: int = 10000, starting_page: int = 0
+) -> None:
+    page = starting_page
+    while True:
+        if ending_page and ending_page == page:
+            break
+
+        offer_ids = collective_offers_repository.get_paginated_active_collective_offer_ids(limit=limit, page=page)
+        if not offer_ids:
+            break
+        search.reindex_offer_ids(offer_ids)
```

Comment:
Typo ici (mauvais copié-collé ?), `reindex_offer_ids()` indexe des offres **individuelles**, pas des offres **collectives**. Je ne trouve pas de fonction synchrone pour ça dans `pcapi.core.search`, mais on peut ici tout à fait appeler l'équivalent asynchrone (ça me semble même être une meilleure idée que d'appeler une fonction synchrone) : 

```suggestion
        search.async_index_collective_offer_ids(collective_offer_ids(offer_ids)
```

Reply:
Oui bien vu !

Reply:
J'ai quand même dû modifier la requête pour récupérer les ids à indexer car dans le `reindex_offer_ids` il y a un tri qui est fait entre les offres à indexer et les offres à désindexer (en se basant sur le `is_eligible_for_search`) 
Si je gardais la fonction de base, on aurait indexé des offres non éligibles à la recherche
Sinon il y a toujours les 2 fonctions synchrones `_reindex_collective_offer_ids` et `_reindex_collective_offer_template_ids` mais tu avais l'air de dire qu'il fallait privilégier la méthode asynchrone ?